sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -538,11 +542,18 @@
             Objects.requireNonNull(receipt, "'receipt' is required and cannot be null.");
             recognizeReceiptsOptions = getRecognizeReceiptOptions(recognizeReceiptsOptions);
             final boolean isFieldElementsIncluded = recognizeReceiptsOptions.isFieldElementsIncluded();
+            final String localeInfo
+                = recognizeReceiptsOptions.getLocale() == null ? "" : recognizeReceiptsOptions.getLocale();
```

Comment:
is it normal for Java to pass empty string instead of null into the request if the user doesn't specify a param?

Reply:
does the service behavior changes if "" is sent instead of not passing anything?

Reply:
>is it normal for Java to pass empty string instead of null into the request if the user doesn't specify a param?

So, Java doesn't need to default to "" when the user doesn't specify. I initially had test recordings with "" so tests were failing. Will update the PR with this change and updated recordings.

>does the service behavior changes if "" is sent instead of not passing anything?

Don't know about the service behavior but the url changes from `includeTextDetails=false&locale=`  when specified ("") to `includeTextDetails=false"` when null.

cc: @kristapratico @maririos

Reply:
yeah I expect the url to change. I haven't done tests here so idk the service behavior but it will be good to validate before making an assumption.
I remember in TA, for example, sending "" for `Language` is different than not sending the `language` parameter

Reply:
Talked offline, currently, the client just passes down the supplied value by the user.